app/models/affiliate.rb

```diff
@@ -7,17 +7,16 @@ class Affiliate < ActiveRecord::Base
   scope :by_name, -> { order(:name) }
 
   def self.OTHER
-    @@other ||= find_or_create_by(name: "Other", subaffiliates_enabled: true)
+    @@other ||= find_or_create_by(name: "Other") { |a| a.subaffiliates_enabled = true }
```

Comment:
No need to change this since there's only the one extra field to be set, but I want to point you to `create_with` in case you need to do something like this in the future with more attributes.

`create_with(subaffiliates_enabled: true).find_or_create_by(name: "Other")` should be equivalent.

http://apidock.com/rails/v4.2.7/ActiveRecord/Relation/find_or_create_by